compiler/onnx-tools/onnx-dump.py

```diff
@@ -120,8 +150,9 @@ def _dump_inputoutputs(onnx_model):
 def _dump_graph(onnx_model):
     _dump_header(onnx_model)
     _dump_operators(onnx_model)
+    vis = _gather_value_infos(onnx_model)
```

Comment:
Why not calling `_gather_value_infos` inside `dump_nodes`? It is called just for `_dump_nodes`.

Reply:
It would be possible to keep the current input signature of `_dump_nodes`.